hw/ip/power_manager/data/power_manager.hjson

```diff
@@ -404,7 +404,7 @@
       swaccess: "rw",
       hwaccess: "hrw",
       fields: [
-        { bits: "0", name: "INTR_STATE", desc: "Interrupt state Reg" }
+        { bits: "31:0", name: "INTR_STATE", desc: "Interrupt state Reg" }
```

Comment:
as above

Reply:
The size may vary depending on the number of external interrupts, so I would leave it as it is, 32 bits.

Reply:
Solved.